arangosh/Utils/ClientManager.cpp

```diff
@@ -54,7 +54,8 @@ arangodb::Result getHttpErrorMessage(arangodb::httpclient::SimpleHttpResult* res
     std::shared_ptr<VPackBuilder> parsedBody = result->getBodyVelocyPack();
     VPackSlice const body = parsedBody->slice();
 
-    int serverCode = VelocyPackHelper::getNumericValue<int>(body, "errorNum", 0);
+    auto serverCode =
+        ErrorCode{VelocyPackHelper::getNumericValue<int>(body, "errorNum", 0)};
     std::string const& serverMessage =
         VelocyPackHelper::getStringValue(body, "errorMessage", "");
```

Comment:
Seeing this, I think this should use `StaticStrings::ErrorNum` and `StaticStrings::ErrorMessage` instead.

Reply:
Let's do this in a different PR.